drafting/jinja-template.j2

```diff
@@ -0,0 +1,18 @@
+package:
+    name: {{ name }}
+    version: {{ version }}
+source:
+    path: .
+build:
+    script: {{ build.command }}
+requirements:
+    host:
+        - python {{ python }}
+        - setuptools {{ setuptools }}
+    run:
+        - python {{ python }}
+        {{ requirements.qiime2 }}
+        {{ requirements.anaconda }}
+test:
+    imports: {{ test.imports }}
+    commands: {{ test.commands }}
```

Comment:
What happens if a qiime2 recipe doesn't have test imports and/or commands? If a recipe doesn't have those options, then the rendered recipe shouldn't have them, either.

Similarly, what if the qiime2 recipe specifies more than one of either of those options?

Reply:
@thermokarst great questions! Here's what my conda recipe file looks like for the following QIIME 2 recipe files:

-QIIME 2 recipe with no test imports/commands:
```
test:
    imports:
    commands:
```
Resulting conda recipe:
```
test:
    imports: None
    commands: None
```

-QIIME 2 recipe with multiple test imports (this is the current recipe file in this repo: **qiime2-recipe.yml**)
```
test:
  imports:
    - q2_cutadapt
    - from qiime2.plugins import cutadapt
  commands:
    - pytest -k q2_cutadapt
```
Resulting conda recipe:
```
test:
    imports: ['q2_cutadapt', 'from qiime2.plugins import cutadapt']
    commands: ['pytest -k q2_cutadapt']
```

Are you wanting to completely remove the test key if nothing is provided in the QIIME 2 recipe file? And is the format for multiple imports/commands okay as-is, or should it not end up as a list (as shown above)?

Reply:
> Are you wanting to completely remove the test key if nothing is provided in the QIIME 2 recipe file?

Yeah, don't you think that would be a bit more clear?

> And is the format for multiple imports/commands okay as-is, or should it not end up as a list (as shown above)?

I think it is still technically valid yaml, but it would probably look a lot nicer formatted as the expanded list style:

```yaml
test:
    imports: 
        - q2_cutadapt
        - from qiime2.plugins import cutadapt
    commands:
        - pytest -k q2_cutadapt
 ```

Reply:
@thermokarst sounds good! I've updated the jinja template to only display the test key (and associated imports/commands keys) if values are present.

re: formatting as an expanded list - i haven't found a straightforward way to do that yet, but will keep working on it!